src/options.c

```diff
@@ -2,6 +2,7 @@
 #include <grp.h>           // NOLINT(llvmlibc-restrict-system-libc-headers)
 #include <pwd.h>           // NOLINT(llvmlibc-restrict-system-libc-headers)
 #include <stdio.h>         // NOLINT(llvmlibc-restrict-system-libc-headers)
+#include <string.h>        // NOLINT(llvmlibc-restrict-system-libc-headers)
```

Comment:
Likewise. strerror() doesn't seem used here.

Reply:
Same as above:
```
/home/teng/src/https_dns_proxy/src/options.c: In function ‘options_parse_args’:
/home/teng/src/https_dns_proxy/src/options.c:128:8: warning: implicit declaration of function ‘strcmp’ [-Wimplicit-funct
ion-declaration]
  128 |       !strcmp(opt->logfile, "-")) {
      |        ^~~~~~
/home/teng/src/https_dns_proxy/src/options.c:136:7: warning: implicit declaration of function ‘strncmp’ [-Wimplicit-func
tion-declaration]
  136 |       strncmp(opt->resolver_url, "https://", 8) != 0) {
      |       ^~~~~~~
/home/teng/src/https_dns_proxy/src/options.c:136:46: warning: ‘strncmp’ argument 3 type is ‘int’ where ‘long unsigned in
t’ is expected in a call to built-in function declared without prototype [-Wbuiltin-declaration-mismatch]
  136 |       strncmp(opt->resolver_url, "https://", 8) != 0) {
      |                                              ^
<built-in>: note: built-in ‘strncmp’ declared here
```

Reply:
Oh, wow. I see now. Looks good to me. Thanks!